card/mod/03_machines/lib/card/machine.rb

```diff
@@ -116,7 +116,8 @@ def self.included(host_class)
         event_suffix = host_class.name.gsub ':', '_'
         host_class.event(
           "reset_machine_output_#{ event_suffix }".to_sym,
-          after: :store_subcards, on: :save
+          after: :expire_related, on: :save
+          #after: :store_subcards, on: :save
```

Comment:
Incorrect indentation detected (column 10 instead of 8).<br>Missing space after #.